frontend/package-lock.json

```diff
@@ -22,7 +22,6 @@
         "@patternfly/react-table": "^5.1.1",
         "@patternfly/react-topology": "^5.1.0",
         "@patternfly/react-virtualized-extension": "^5.0.0",
-        "@storybook/test-runner": "^0.16.0",
```

Comment:
Is this intentional? Pretty weird if we have change something in the package json that the package-lock didn't get it.

Reply:
I don't think you should ever modify the package-lock like this... What's the solution being applied here?

Reply:
This wasn't intentional, thanks for catching it. I just ran `npm install` and it snuck in for some reason. It looks like rebasing and re-running install has undone it.

Reply:
This is what I get for using `git commit -a` 🙃